curriculum/challenges/english/25-front-end-development/lecture-html-fundamentals/67083868d5fdcb17bf8c14bd.md

```diff
@@ -2,7 +2,7 @@
 id: 67083868d5fdcb17bf8c14bd
 title: What Are HTML Entities, and What Are Some Common Examples?
 challengeType: 11
-videoId: VJmx-XzTqg8
+videoId: Z9C18t3fXt0
```

Comment:
I don't think this is the right video. The title says `DO NOT USE`

Reply:
yeah, that is the one that had errors and is being redone. 
For the `What are HTML entities and what are some common examples?` and `What is the difference between slashes, a single dot, and double dot in path syntax?` lectures, let's just add transcripts here and challenge type since we don't know when those will be ready.

If we are able to get the real videos great, otherwise at least we have transcripts for now.

Reply:
@moT01 

It looks like the path syntax video was just added 

https://www.youtube.com/watch?v=GTgzy4wPXcM

Reply:
✅

Reply:
I added the "What is the difference between slashes..." one - I think Ania is working on that last one now.